common/components/stores/ProjectSearchStore.js

```diff
@@ -123,8 +139,9 @@ class ProjectSearchStore extends ReduceStore<State> {
   }
 
   _updateFindProjectArgs(state: State): State {
+    let findProjectsArgs: FindProjectsArgs;
     if(state.projectsData && state.projectsData.allTags) {
-      const findProjectsArgs: FindProjectsArgs = _.pickBy({
+      findProjectsArgs = _.pickBy({
```

Comment:
Why remove the const and type specification?

Reply:
? No idea. I don't recall doing that.